zbus::fdo::ManagedObjects objects can not be cloned

The following program does not compile:

```rust
use zbus::blocking::fdo::ObjectManagerProxy;
use zbus::blocking::Connection;
use std::error::Error;

fn main() -> Result<(), Box<dyn Error>> {
    let conn = Connection::system()?;
    let obj_manager = ObjectManagerProxy::new(
        &conn,
        "org.freedesktop.UDisks2",
        "/org/freedesktop/UDisks2",
    )?;
    let all_objects = obj_manager.get_managed_objects()?;
    let cloned = all_objects.clone();
    Ok(())
}
```

Yielding instead:
```
error[E0599]: the method `clone` exists for struct `HashMap<OwnedObjectPath, HashMap<OwnedInterfaceName, HashMap<String, OwnedValue>>>`, but its trait bounds were not satisfied
  --> src/main.rs:13:30
   |
13 |     let cloned = all_objects.clone();
   |                              ^^^^^ method cannot be called due to unsatisfied trait bounds
   |
   = note: the following trait bounds were not satisfied:
           `HashMap<OwnedInterfaceName, HashMap<std::string::String, OwnedValue>>: Clone`
           which is required by `HashMap<OwnedObjectPath, HashMap<OwnedInterfaceName, HashMap<std::string::String, OwnedValue>>>: Clone`
```

I believe this is because `OwnedValue` does not implement the `Clone` trait, providing instead a `try_clone()` method.

The [zbus::fdo::ManagedObjects](https://docs.rs/zbus/latest/zbus/fdo/type.ManagedObjects.html) is an alias to `HashMap<OwnedObjectPath, HashMap<OwnedInterfaceName, HashMap<String, OwnedValue>>>`. 


zbus version: `4.4.0`

--- a/zvariant/src/owned_value.rs
+++ b/zvariant/src/owned_value.rs
@@ -283,6 +283,19 @@ impl<'de> Deserialize<'de> for OwnedValue {
     }
 }
 
+impl Clone for OwnedValue {
+    /// Clone the value.
+    ///
+    /// # Panics
+    ///
+    /// This method can only fail on Unix platforms for [`Value::Fd`] variant containing an
+    /// [`Fd::Owned`] variant. This happens when the current process exceeds the limit on maximum
+    /// number of open file descriptors.
+    fn clone(&self) -> Self {
+        Self(self.0.clone())
+    }
+}
+
 #[cfg(test)]
 mod tests {
     use std::{collections::HashMap, error::Error};

--- a/zvariant/src/value.rs
+++ b/zvariant/src/value.rs
@@ -974,6 +974,20 @@ impl<'a> TryFrom<&Value<'a>> for Value<'a> {
     }
 }
 
+impl Clone for Value<'_> {
+    /// Clone the value.
+    ///
+    /// # Panics
+    ///
+    /// This method can only fail on Unix platforms for [`Value::Fd`] variant containing an
+    /// [`Fd::Owned`] variant. This happens when the current process exceeds the limit on maximum
+    /// number of open file descriptors.
+    fn clone(&self) -> Self {
+        self.try_clone()
+            .expect("Process exceeded limit on maximum number of open file descriptors")
+    }
+}
+
 #[cfg(test)]
 mod tests {
     use std::collections::HashMap;

